deploy/tekton/tasks/execute_sast_ai_workflow.yaml

```diff
@@ -75,12 +75,29 @@ spec:
       type: string
       description: "Optional GDrive SA file name"
       default: "service_account.json"
+    - name: GCS_BUCKET_NAME
+      type: string
+      description: "GCS bucket name for uploading SARIF reports (optional)"
+      default: ""
+    - name: GCS_BUCKET_UPLOAD_ENABLED
```

Comment:
I think this is a redundant variable. If someone set the `GCP_BUCKET_NAME`, they hardly want it to be disabled.